### PR TITLE
Add meetings notes for 2019-10-30

### PR DESCRIPTION
**Pull request by jkrems**
Meeting issue: https://github.com/nodejs/modules/issues/408

### CODE REVIEW

**Comment by robpalme** [comment]
Thank you for publishing the minutes so quickly.

File: doc/meetings/2019-10-30.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+## Present
+
+* Michael Zasso (@targos)
+* Jan Krems (@jkrems)
+* Geoffrey Booth (@GeoffreyBooth)
+* Guy Bedford (@guybedford)
+* Myles Borins (@MylesBorins)
+* Saleh Abdel Motaal (@SMotaal)
+* Jordan (@ljharb)
+* Sven Sauleau (@xtuc)
```
Comment (weswigham) [inline doc/meetings/2019-10-30.md]:
```suggestion
* Sven Sauleau (@xtuc)
* Wesley Wigham (@weswigham)
```

File: doc/meetings/2019-10-30.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+* Guy: I’d be very tempted to not document the node condition. I’m of the opinion that the node mono-culture won’t last and there’s a future of many different runtimes. While it is the mono-culture, default should be generally used. I wouldn’t remove the node condition.
+* Jan: I think modules using node APIs should generally call themselves out, to guard their behavior. Default should generally be for code that’s portable.
+* Jordan: If I want to import(pkg/feature), how do I designate what to use in the conditional expression? Believe node should exist.
+* Myles: For sake of time, we don’t have time to work through final list of conditions that should be supported by node. When we try to reach consensus, we could reach consensus on behaviors without locking down these details. Perhaps for the consensus better to focus on high level requirements and capabilities.
+* Jan: Should consider switching back to require guard. Default should be portable and CJS isn’t portable.
+* Guy: We can investigate, don’t need to lock this down (to Myles’ point above).
+
+### Reaching Consensus
+
```
Comment (MylesBorins) [inline doc/meetings/2019-10-30.md]:
```suggestion

### Topics to reach consensus on

```

File: doc/meetings/2019-10-30.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  - Blocks unflagging?
+  - Conditional Mapping
+  - Alternative entry points a single specifier within node.js runtime
+* Unflagging
+  - What features need to be unflagged first?
+  - Dual mode modules blocking unflagging?
+  - Timeline
+- Ways to move forward
+- Continuing activity of the group
+
```
Comment (MylesBorins) [inline doc/meetings/2019-10-30.md]:
```suggestion

### Consensus reached with quorum of voting members present

```

File: doc/meetings/2019-10-30.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+* Guy: The longer we don’t unflag, the more people are trying to adjust but can’t get proper feedback on how well their setups work. With 12 LTS missed, there’s questions about our progress. Would like to unflag as experimental to get more feedback.
+* Myles: Worth considering that node 14 is heading for LTS, cut in April. If we want to stabilize in 14, we should have everything unflagged ahead of that branch cut.
+* Myles: Any objections to:
+  - Land conditional exports behind additional flag.
+  - Document hazard around singleton.
+  - Unflag modules with that state in node 13. This preserves the experimental warning.
+  - At the end of January, if there’s no better alternative for dual packages, we will remove the conditional exports flag and turn the feature on by default.
+* Wes: Clarifying that we’ll still discuss in a meeting before removing the flag.
+* Myles: Clarifying that January is a deadline. We can make decisions earlier with consensus.
```
Comment (MylesBorins) [inline doc/meetings/2019-10-30.md]:
```suggestion
Myles: Clarifying that end of January is a deadline. We can make decisions earlier with consensus.
```

Comment (GeoffreyBooth) [reply doc/meetings/2019-10-30.md]: Rewriting history! 😉

**Pull request merged**